aten/src/ATen/core/ivalue.h

```diff
@@ -705,7 +708,12 @@ struct C10_EXPORT ivalue::Object final : c10::intrusive_ptr_target {
     return typename_;
   }
 
+  Symbol name_space() const {
+    return namespace_;
+  }
+
  private:
+  const Symbol namespace_;
```

Comment:
`namespace_` but `name_space`?

Reply:
"namespace" is a reserved word so I did all sorts of awkward things to avoid it. `name_space_` feels silly, but I guess it is consistent with `name_space` 😞